z02-parts-of-a-graph.md

```diff
@@ -130,6 +130,8 @@ often the data is homogenous arrays.
       {% include examples/data.js %}
 d3.min(numbers);
 // 1
+d3.extent(numbers);
```

Comment:
Extent is used an an example a few lines below. Should we remove that to move it up here?